tests/conftest.py

```diff
@@ -138,7 +138,7 @@ def softfork_height(request) -> int:
     return request.param
 
 
-saved_blocks_version = "rc5"
+saved_blocks_version = "2.0"
```

Comment:
are we changing the format of something here? like FullBlocks?

Reply:
`BlockTools` produce with high probability blocks failing the new plot_id condition (hence making tests fail). Working to rebuild the block cache with the new `BlockTools` from this PR.